### PR TITLE
fix: fix tx db changed subject and add tx in event

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/models/subjects/tx-db-changed-subject.ts [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+
+export interface EventAndTx {
+  event: string
+  tx: Transaction
+}
 
 // subscribe this Subject to monitor any transaction table changes
 export class TxDbChangedSubject {
-  static subject = new ReplaySubject<string>(100)
+  static subject = new ReplaySubject<EventAndTx>(100)
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/models/subjects/tx-db-changed-subject.ts]: The type name is weird

Comment (Keith-CY) [reply packages/neuron-wallet/src/models/subjects/tx-db-changed-subject.ts]: How about `TransactionMessage`

Comment (classicalliu) [reply packages/neuron-wallet/src/models/subjects/tx-db-changed-subject.ts]: any advice

Comment (classicalliu) [reply packages/neuron-wallet/src/models/subjects/tx-db-changed-subject.ts]: Renamed to `TransactionChangedMessage`

**Pull request merged**